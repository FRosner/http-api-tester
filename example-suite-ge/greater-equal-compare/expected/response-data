{
  "b" : [1, 2],
  "a" : 24
}
