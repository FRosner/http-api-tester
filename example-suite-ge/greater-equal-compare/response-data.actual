{
  "a" : 42,
  "b" : [1, 2]
}
