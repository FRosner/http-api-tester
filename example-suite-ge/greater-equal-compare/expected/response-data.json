[
  {
    "id": "http-server",
    "description": "A simple Python HTTP request handler. This class serves files from the current directory and below, directly mapping the directory structure to HTTP requests.",
    "parameters": [
      "id"
    ]
  },
  {
    "id": "jupyter",
    "description": "Open source, interactive data science and scientific computing across over 40 programming languages.",
    "parameters": [
      "id"
    ]
  },
  {
    "id": "zeppelin",
    "description": "A web-based notebook that enables interactive data analytics.\nYou can make beautiful data-driven, interactive and collaborative documents with SQL, Scala and more.",
    "parameters": [
      "id"
    ]
  }
]
